src/System.AppContext/src/System.AppContext.csproj

```diff
@@ -22,7 +22,8 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net463_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net463_Release|AnyCPU'" />
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)'=='true'">
-    <TargetingPackReference Include="mscorlib" />
+    <TargetingPackReference Include="mscorlib" Condition="'$(TargetGroup)' == 'net463'" />
```

Comment:
When I remove the TargetingPackReference for System.Private.CoreLib, it does not build anymore. 

Removing TargetingPackReference for mscorlib works fine - I will do that.